### DISCUSSION TITLE
Rounded Edges Frameless App Mac?

### DISCUSSION THREAD

**Discussion by Debdut** [wailsapp/wails#1804] [Q&A] [answered]
I'm getting non-rounded edges for frameless app on mac, how to get rounded edges?

<img width="846" alt="Screen Shot 2022-08-30 at 1 47 46 PM" src="https://user-images.githubusercontent.com/7561070/187386549-db1397c2-9a50-4204-990e-4bb85461bdf8.png">

**Comment by leaanthony** [comment, marked as answer]
Please file a bug 🙏
